service-runner/src/main/resources/profiles/default/config.env

```diff
@@ -34,5 +34,5 @@ sep12.enabled=true
 sep31.enabled=true
 sep38.enabled=true
 sep24.enabled=true
-sep24.interactive_url.base_url=http://localhost:8081/sep24/interactive
-sep24.more_info_url.base_url=http://localhost:8081/sep24/transaction/more_info
+sep24.interactive_url.base_url=http://localhost:3000/
```

Comment:
IIRC default config is used in integration tests too, it will break them.
Considering UI is not needed for tests, can you just create a new profile for that? (Ideally with IDEA .run configuration)

Reply:
The integration tests that depend on SEP24 interactive URLs use the `sep24` profile. I didn't modify those URLs for this reason.

I discussed this with Jamie last week but we decided to just spin up the UI as part of the dev stack. Introducing a separate run configuration would require some changes to the service runner since we need to introduce a way to optionally start the UI in docker compose. I'm in favor of just spinning it up in the dev stack. What do you think?

Reply:
Good point.
I agree, it can be a part of a dev stack, and I agree that making it optional sounds like a good idea.